MI-Xib_cell/Views/CustomTableViewCell.swift

```diff
@@ -8,23 +8,23 @@
 import UIKit
 
 class CustomTableViewCell: UITableViewCell {
-
+    
     @IBOutlet weak var commonView: UIView!
     @IBOutlet weak var roleLabel: UILabel!
     @IBOutlet weak var nameLabel: UILabel!
     
     static var reuseId: String = "customCell"
-    private var labelData: [LabelData] = []
+    
+    var labelData: [LabelData] = []
```

Comment:
Why you take there [LabelData] not LabelData?

Reply:
I will try like you say with LabelData?

Reply:
you have an model LabelData, if you will get the variable with name labelsData i will say to use, that you should send labelData, but you have an name labelData, idk how to say to you, that you should send single object of you array of data

Reply:
I took the array in cell file and changed the function.